main.go

```diff
@@ -37,7 +37,7 @@ func main() {
 	opts := &plugin.ServeOpts{ProviderFunc: provider.New(version)}
 
 	if debugMode {
-		err := plugin.Debug(context.Background(), "registry.terraform.io/smartx/cloudtower", opts)
+		err := plugin.Debug(context.Background(), "registry.terraform.io/yuyz0112/cloudtower", opts)
```

Comment:
should be smartx ?

Reply:
It need change makefile to fix the path.